CMakeLists.txt

```diff
@@ -27,6 +28,11 @@ if (ANDROID)
   set(LVK_WITH_SAMPLES OFF)
 endif()
 
+if(LVK_WITH_WAYLAND AND (ANDROID OR APPLE OR WIN32))
+  message(STATUS "WARNING: LVK_WITH_WAYLAND=ON can be set only on Linux")
```

Comment:
I think it should be `FATAL_ERROR` instead of `STATUS`. Because this means the user is doing something completely wrong.